observability/network_observability/network-observability-operator-release-notes.adoc

```diff
@@ -13,6 +13,61 @@ These release notes track the development of the Network Observability Operator
 
 For an overview of the Network Observability Operator, see xref:../../observability/network_observability/network-observability-overview.adoc#dependency-network-observability[About Network Observability Operator].
 
+[id="network-observability-operator-release-notes-1-6"]
+== Network Observability Operator 1.6.0
+The following advisory is available for the Network Observability Operator 1.6.0:
+// note to self: update errata live ID
+* link:https://access.redhat.com/errata/RHSA-2024:0853[Network Observability Operator 1.6.0]
+
+[id="network-observability-operator-1.6.0-features-enhancements"]
+=== New features and enhancements
+
+[id="network-observability-custom-metrics-1.6"]
+==== Custom metrics API
+You can create custom metrics out of the flowlogs data using the `FlowMetrics` API. Flowlogs data can be used with Prometheus labels to customize cluster information on your dashboards. For more information, see xref:../../observability/network_observability/metrics-alerts-dashboards.adoc#network-observability-custom-metrics_metrics-dashboards-alerts[Custom metrics] and <FLOW_METRICS_API_REF_LINK_HERE>
+
+[id="network-observability-eBPF-performance-enhancements-1.6"]
+==== eBPF performance enhancements
```

Comment:
I think we need first to say that we improved the performances, both in terms of CPU and memory, of the agent. And then as a side thing that we also introduced new health metrics for the agent, visible in the health dashboard, as well as this alert on dropped flows.

Reply:
Thank you!